### ISSUE TITLE
Extreme CPU Usage with v1.0.4

### ISSUE THREAD

**Issue by Luk-ESC** [Artillex-Studios/AxMinions#5]
According to Spark, AxMinions is taking up ~20% of our server thread:
![image](https://github.com/Artillex-Studios/AxMinions/assets/55601315/f8daf3c3-0ee6-4f73-a9e4-e02aac5f390d)

# Info
/axminions stats:
![image](https://github.com/Artillex-Studios/AxMinions/assets/55601315/18436cd9-7899-459f-9ef4-545737fc5054)

AxMinions Version: 1.0.4

```
The platform is a Bukkit server running version "git-Purpur-2062 (MC: 1.20.1)".
The detected Minecraft version is "1.20.1".
The server is running in online mode (BungeeCord).
The server had a player count of 120 when the profile completed.
The system is running Ubuntu 22.04.3 LTS (amd64) version "5.10.0-28-amd64" and has 5 CPU threads available.
The CPU is described as an Intel(R) Core(TM) i9-9900K CPU @ 3.60GHz.
The process is using Java 17.0.10 (Temurin-17.0.10+7 from Eclipse Adoptium).
The current process uptime is 7h 33m 19s.
The profiler was running for 2h 1s (127957 ticks).
```
---

Is this to be expected?

Ready to provide more debug info.

**Comment by AverageGithub** [comment]
Could you try updating the plugin? I'm pretty sure I have fixed this in one of the versions since 1.0.4

**Comment by Luk-ESC** [comment]
Alright upgrading to v1.0.7 seems to have fixed it! Now down to ~0.64% after 15m of running. Thanks for your help!
